src/test/java/core/basesyntax/strategy/AddOperationHandlerTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.ShopOperation;
+import core.basesyntax.model.Fruit;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AddOperationHandlerTest {
+    private AddOperationHandler addOperationHandler;
+    private Fruit banana = new Fruit("banana");
+
+    @After
+    public void tearDown() {
+        Storage.storage.clear();
+    }
+
+    @BeforeClass
+    public static void beforeClass() {
+        Storage.storage.clear();
+    }
+
+    @Before
+    public void setUp() {
+        addOperationHandler = new AddOperationHandler();
+    }
+
```

Comment:
Do we really need to initialize a new handler before every test?

Reply:
yes, it is used in every methods, and we can't initialize it in  @beforeClass method

Reply:
let's make it static and initialize before all methods